--- a/ddclient.in
+++ b/ddclient.in
@@ -29,7 +29,7 @@ my $program   = $programd;
     return $subst;
 }
 
-my $etc = subst_var('@sysconfdir@', '/etc/ddclient');
+my $etc = subst_var('@sysconfdir@', '/etc');
 my $cachedir = subst_var('@localstatedir@', '/var') . '/cache/ddclient';
 my $savedir = '/tmp';
 if ($program =~ /test/i) {
--- a/ddclient.conf.in
+++ b/ddclient.conf.in
@@ -22,7 +22,7 @@ daemon=300				# check every 300 seconds
 syslog=yes				# log update msgs to syslog
 mail=root				# mail all msgs to root
 mail-failure=root		# mail failed update msgs to root
-pid=@runstatedir@/ddclient.pid		# record PID in file.
+pid=@runstatedir@/ddclient/ddclient.pid		# record PID in file.
 ssl=yes					# use ssl-support.  Works with
 						# ssl-library
 # postscript=script		# run script after updating.  The

