--- ddclient-3.8.2/ddclient~	2013-12-26 10:04:25.000000000 +0100
+++ ddclient-3.8.2/ddclient	2014-03-12 19:31:43.839475256 +0100
@@ -29,7 +29,7 @@ my $program   = $programd;
 $program  =~ s/d$//;
 my $now       = time;
 my $hostname  = hostname();
-my $etc       = ($program =~ /test/i) ? './'   : '/etc/ddclient/';
+my $etc       = ($program =~ /test/i) ? './'   : '/etc/';
 my $cachedir  = ($program =~ /test/i) ? './'   : '/var/cache/ddclient/';
 my $savedir   = ($program =~ /test/i) ? 'URL/' : '/tmp/';
 my $msgs      = '';
--- ddclient-3.8.2/sample-etc_ddclient.conf~	2013-12-26 10:04:25.000000000 +0100
+++ ddclient-3.8.2/sample-etc_ddclient.conf	2014-03-12 18:58:12.719452850 +0100
@@ -22,7 +22,7 @@ daemon=300				# check every 300 seconds
 syslog=yes				# log update msgs to syslog
 mail=root				# mail all msgs to root
 mail-failure=root			# mail failed update msgs to root
-pid=/var/run/ddclient.pid		# record PID in file.
+pid=/var/run/ddclient/ddclient.pid	# record PID in file.
 ssl=yes					# use ssl-support.  Works with
 					# ssl-library
 # postscript=script			# run script after updating.  The
